docs/workflow/implementing/messaging/microsurveys/ios-mobile-microsurveys.md

```diff
@@ -0,0 +1,162 @@
+---
+id: ios-microsurveys
+title: Microsurvey Feature for iOS
+slug: /messaging/microsurveys/ios
+---
+## Introduction
+This page provides context on the microsurvey feature and a guide to configure the recipes for future microsurveys, specifically on the iOS platform. The microsurvey feature is an extension of the [messaging framework](/messaging/mobile-messaging), and allows us to capture user’s sentiment on any specific app’s features such as the homepage.
+
+Note that this page was created during the MVP phase of the microsurvey project.
+
+## UI
+The microsurvey feature is composed of two main UI components. One is the prompt and one is the actual survey.
+
+### Prompt
+
+The prompt is an invitation to start the survey and it is shown when the triggers are met. The triggers are determined through the mobile messaging infrastructure and should be specified by the Nimbus recipe. For testing, we have created a trigger in which the user will be eligible to receive the survey after visiting the homepage twice.
+
+The prompt is shown until the user completes the survey, closes the survey or reached the maximum number of display specified in the mobile messaging configurations.
+
+<img src="/img/messaging/microsurveys/ios-prompt.png" alt="Prompt view" className="img-sm"/>
+<br/><br/>
+
+1. This title of the prompt which can be customized, but also has a fallback string.
+2. The close button dismisses the survey and it won't be shown again.
+3. The continue button starts the survey and shows the survey sheet. It can be customized, but also has a fallback string.
```

Comment:
There's a fallback string for the entire survey?  Also what is it?

Reply:
This is for the button label, I updated to clarify.